File: src/main/java/org/sdi/annotations/Inject.java
package org.sdi.annotations;

import java.lang.annotation.*;

/**
 * @author Luis Miguel Barcos
 */
@Retention(RetentionPolicy.RUNTIME)
@Target({ElementType.CONSTRUCTOR, ElementType.FIELD})
@Documented
public @interface Inject {
    public String value() default "";
}

File: src/main/kotlin/org/sdi/annotations/Inject.kt
package org.sdi.annotations

import kotlin.annotation.AnnotationRetention.RUNTIME
import kotlin.annotation.AnnotationTarget.*

/**
 * @author Luis Miguel Barcos
 */
@Retention(RUNTIME)
@Target(CONSTRUCTOR, FIELD)
@MustBeDocumented
annotation class Inject(val value: String = "")
